.rubocop.yml

```diff
@@ -21,8 +21,17 @@ AllCops:
     - 'lib/log_stuff.rb'
     - 'config/routes.rb'
     - 'Gemfile'
-    - 'spec/**/*'
```

Comment:
How about to keep this and use the `Include:` to include specific files/folders if needed?
```
Include:
  - 'spec/foo.rb'
  - 'spec/bar/**/*'
```